src/modules/esc_battery/EscBattery.hpp

```diff
@@ -73,5 +75,6 @@ class EscBattery : public ModuleBase<EscBattery>, public ModuleParams, public px
 	uORB::Subscription _actuator_ctrl_0_sub{ORB_ID(actuator_controls_0)};
 	uORB::SubscriptionCallbackWorkItem _esc_status_sub{this, ORB_ID(esc_status)};
 
+	static constexpr uint32_t ESC_BATTERY_INTERVAL_US = 10_ms; // assume higher frequency esc feedback than 100Hz
```

Comment:
@cmic0 is that satisfied in your use-case?

Reply:
AFAIK The ESC status gets updated with 250Hz and that's then how fast `EscBattery` would update if not throttled to 100Hz.

Reply:
I checked myself and it's apparently not:
![image](https://user-images.githubusercontent.com/4668506/82300876-bb614380-99b7-11ea-9cc7-1e81b7291c12.png)
So I'll have to lower the rate.

Reply:
Done in https://github.com/PX4/Firmware/pull/14551/commits/8fe070b703ff21d3c24a1447216d38e3f7f878e4#diff-2c9d42513d81aa1477a9e9c27cbede98R78 after rebasing on master